README.md

```diff
@@ -1,10 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] It would be nicer to add a description of your project "Book store" so others know more about your project.

Reply:
Thanks, @medaminedev66 for suggesting these changes. It's solved.